### DISCUSSION TITLE
Re-routing the z-wave network

### DISCUSSION THREAD

**Discussion by ivdimitro** [zwave-js/zwave-js#1548] [Q&A] [answered]
Hello, I moved my controller and now the routes of the network are not optimal for the location of the controller. Is there a way to force the network to recalculate the routes or if possible to manually assign them? 
From what I've read the only option will be send request to problematic node with "explore frame". Is zwave-js supporting this option?

**Comment by AlCalzone** [comment, marked as answer]
Try this: https://zwave-js.github.io/node-zwave-js/#/api/controller?id=beginhealingnetwork
That should do what you're asking.

The explorer frame option is also enabled by default.

**Comment by ivdimitro** [comment]
I tried the Healing option and it helped a bit. I just takes some time for the network to adapt. A "sleeping" still is unreliable in its reporting but I guess I just have to add a mains powered node nearby.
